test/new_relic/agent/system_info_test.rb

```diff
@@ -324,6 +324,46 @@ def test_system_info_bsd_predicate
     refute_predicate NewRelic::Agent::SystemInfo, :bsd?, 'Did not expect OS to match bsd'
   end
 
+  def test_system_info_windows_predicate
+    NewRelic::Agent::SystemInfo.stubs(:ruby_os_identifier).returns('mingw32')
+
+    assert_predicate NewRelic::Agent::SystemInfo, :windows?, 'Expected OS to match windows'
+
+    NewRelic::Agent::SystemInfo.stubs(:ruby_os_identifier).returns('darwin13')
```

Comment:
I think the 2 halves of this test would be better off living in 2 separate tests. Perhaps:

- test_system_info_windows_is_true
- test_system_info_windows_is_false

Reply:
@fallwith I have made this test similar to ones already present for linux and mac. If you still want it to be separated out, I can do.

Reply:
Ok. I was only looking at the new code from the GitHub diff view.